README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
While I agree this is clearer as to what is happening, the other command @ljharb added helps us specify version numbers. This command ultimately just installs the latest version of the package and not the supported version. This becomes a problem with (a) semver-major changes if we haven't audited them yet and (b) bugs in newer releases (like when moment released 1.15.0 and it broke all our tests). I would prefer to leave the slightly more confusing and more correct command here.